docs/index.md

```diff
@@ -1,37 +1,42 @@
-## Welcome to GitHub Pages
+---
```

Comment:
Isn't this page repeats what is seen in the README ?
These should come in different commits, it will be less confusing.

Reply:
I have do a `ln -s ../README.md` so now we can use it as the index page.